### DISCUSSION TITLE
Another question from me!

### DISCUSSION THREAD

**Discussion by ghost** [TomSchimansky/CustomTkinter#535] [General]
How can I hide certain elements when button_1 is clicked, and show them when button_2 is clicked.

**Comment by sumeshir26** [comment]
Try using .pack_forget()  to hide and .pack() to show again

**Comment by Akascape** [comment]
| **Show** | **Hide** |
| -----------| --------- |
| widget.grid() | widget.grid_forget() |
| widget.pack() | widget.pack_forget() |
| widget.place() | widget.place_forget() |

**Reply by ghost** [reply]
Alright, thanks y'all
I really appreciate it. ❤️
